test/testdrive/kafka-sink-errors.td

```diff
@@ -158,54 +158,6 @@ contains:creating admin client failed: Client config error: Invalid value "FOO"
   FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}'
 contains:Meta data fetch error: BrokerTransportFailure (Local: Broker transport failure)
 
-! CREATE SINK invalid_ssl_certificate_location FROM v1
-  INTO KAFKA BROKER '${testdrive.kafka-addr}'
-  TOPIC 'testdrive-kafka-sink-errors-${testdrive.seed}'
-  WITH (security_protocol = SSL , ssl_certificate_location = 'foo')
-  FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}'
-contains:Invalid WITH option ssl_certificate_location='foo': file does not exist
-
-> CREATE SECRET invalid_ssl_certificate_location_secret AS 'foo'
-
-! CREATE SINK invalid_ssl_certificate_location FROM v1
-  INTO KAFKA BROKER '${testdrive.kafka-addr}'
-  TOPIC 'testdrive-kafka-sink-errors-${testdrive.seed}'
-  WITH (security_protocol = SSL , ssl_certificate_location = SECRET invalid_ssl_certificate_location_secret)
-  FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}'
-regex:Invalid WITH option ssl_certificate_location=.*: file does not exist
-
-! CREATE SINK invalid_ssl_key_location FROM v1
-  INTO KAFKA BROKER '${testdrive.kafka-addr}'
-  TOPIC 'testdrive-kafka-sink-errors-${testdrive.seed}'
-  WITH (security_protocol = SSL , ssl_key_location = 'foo')
-  FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}'
-contains:Invalid WITH option ssl_key_location='foo': file does not exist
-
-> CREATE SECRET invalid_ssl_key_location_secret AS 'foo'
-
-! CREATE SINK invalid_ssl_key_location FROM v1
-  INTO KAFKA BROKER '${testdrive.kafka-addr}'
-  TOPIC 'testdrive-kafka-sink-errors-${testdrive.seed}'
-  WITH (security_protocol = SSL , ssl_key_location = SECRET invalid_ssl_key_location_secret)
-  FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}'
-regex:Invalid WITH option ssl_key_location=.*: file does not exist
```

Comment:
Part of this test was validating that using a secret and using a plaintext value both returned errors that were interpretable.

The reason that `extract` took a `secrets_reader` before was so that for the old `Config::Path` types, we had the same error -- rather than a bad secret giving a mysterious ssl error.

Can we have some sort of test about the basic form of errors?  Not sure if there's a structural reason these tests got axed? are they no longer applicable?

Reply:
We're removing the path-like options from cloud, where they're not usable. I'm currently working on building out tests for more of the option handling, though file locations won't be part of the work.

Reply:
yup -- I had clocked that paths were no longer applicable -- but good to hear that some tests for options handling is incoming!